services/pull/patch.go

```diff
@@ -283,18 +283,9 @@ func checkConflicts(ctx context.Context, pr *models.PullRequest, gitRepo *git.Re
 	}
 
 	if !conflict {
-		var treeHash string
-		treeHash, _, err = git.NewCommand(ctx, "write-tree").RunStdString(&git.RunOpts{Dir: tmpBasePath})
-		if err != nil {
-			return false, err
-		}
-		treeHash = strings.TrimSpace(treeHash)
-		baseTree, err := gitRepo.GetTree("base")
-		if err != nil {
-			return false, err
-		}
-		if treeHash == baseTree.ID.String() {
-			log.Debug("PullRequest[%d]: Patch is empty - ignoring", pr.ID)
+		log.Info("PullRequest[%d]: Head SHA: %s, Merge base SHA: %s", pr.ID, pr.HeadCommitID, pr.MergeBase)
```

Comment:
I presume this is just meant to be here whilst your testing... 

Please don't forget to remove this.

Reply:
fixed :) I reduced the log level to debug (if it is ok).